openquake/hazardlib/valid.py

```diff
@@ -533,7 +533,8 @@ def positivefloats(value):
     :returns:
         a list of positive floats
     """
-    floats = list(map(positivefloat, value.split()))
+    values = value.lstrip('[').rstrip(']').split()
```

Comment:
`value.strip('[]').split()` should work too

Reply:
Indeed